src/MUIDataTable.js

```diff
@@ -29,19 +29,28 @@ const defaultTableStyles = theme => ({
   tableRoot: {
     outline: 'none',
   },
+  responsiveBase: {
+    overflowX: 'auto',
```

Comment:
Looks like a duplicate property.

overflow, overflowX

Reply:
It's the base responsive class for the vertical, standard, and simple options. It takes the place of classes like responsiveScroll and responsiveScrollMaxHeight.

Reply:
```css
responsiveBase: {
    **overflowX: 'auto',**
    **overflow: 'auto',**
  },
```

Reply:
oooh, I see. Yeah, that should be a removed - good eye! Should be gone now.